controls/cis_ocp_1_4_0/section-4.yml

```diff
@@ -47,6 +47,8 @@ controls:
       rules:
         - file_groupowner_kubelet_conf
         - file_owner_kubelet_conf
+        #- file_groupowner_kubelet
+        - file_owner_kubelet
```

Comment:
@Vincent056 Are these rules still valid? We have `file_owner_kubelet` that seems reasonable, but `file_groupowner_kubelet` is missing despite having been used in the original CIS profile. Should we drop the rules or re-add the group one?

Reply:
it looks like we were missing that one, let's keep this one here, and re-add the group one